railties/lib/rails/generators/rails/app/templates/config/secrets.yml

```diff
@@ -22,9 +24,3 @@ development:
 
 test:
   secret_key_base: <%= app_secret %>
-
-# Do not keep production secrets in the repository,
-# instead read values from the environment.
-
-production:
-  secret_key_base: <%%= ENV["SECRET_KEY_BASE"] %>
```

Comment:
Should probably replace this with a pointer to encrypted secrets. Maybe something like:

```
# Do not keep production secrets in the unencrypted secrets file.
# Instead, use bin/rails secrets:setup to configure encrypted secrets.
# This is where the secret_key_base should go.
```